package.json

```diff
@@ -28,7 +28,6 @@
     "@angular/platform-browser-dynamic": "7.x",
     "ng-packagr": "5.x",
     "rxjs": "6.x",
-    "rxjs-tslint": "0.1.x",
```

Comment:
This was necessary for the migration. It contained scripts to migrate `rxjs` to v6.

It's no longer needed. This package only includes `tslint` rules, and this project isn't using `tslint` at all. (I do think it _should_ be using a linter, but that's out of the scope of this PR. If we do eventually add a linter, it'll likely be `eslint` since `tslint` was deprecated a while ago.)

(We weren't really using rxjs in this Repo at all, so the scripts weren't really needed. But that's not really important.)